test/Microsoft.TestPlatform.Utilities.UnitTests/CodeCoverageRunSettingsProcessorTests.cs

```diff
@@ -0,0 +1,308 @@
+            @"         <ModulePath>.*bcryptPrimitives.dll$</ModulePath>",
+            @"       </Exclude>",
+            @"    </ModulePaths>",
+            @"    <UseVerifiableInstrumentation>True</UseVerifiableInstrumentation>",
+            @"    <AllowLowIntegrityProcesses>True</AllowLowIntegrityProcesses>",
+            @"    <CollectFromChildProcesses>True</CollectFromChildProcesses>",
+            @"    <CollectAspDotNet>false</CollectAspDotNet>",
+            @"    <SymbolSearchPaths/>",
+            @"    <Functions>",
+            @"      <Exclude>",
+            @"        <Function>^std::.*</Function>",
+            @"        <Function>^ATL::.*</Function>",
+            @"        <Function>.*::__GetTestMethodInfo.*</Function>",
+            @"        <Function>.*__CxxPureMSILEntry.*</Function>",
+            @"        <Function>^Microsoft::VisualStudio::CppCodeCoverageFramework::.*</Function>",
+            @"        <Function>^Microsoft::VisualStudio::CppUnitTestFramework::.*</Function>",
+            @"        <Function>.*::YOU_CAN_ONLY_DESIGNATE_ONE_.*</Function>",
+            @"        <Function>^__.*</Function>",
+            @"        <Function>.*::__.*</Function>",
+            @"      </Exclude>",
+            @"    </Functions>",
+            @"    <Attributes>",
+            @"      <Exclude>",
+            @"        <Attribute>^System.Diagnostics.DebuggerHiddenAttribute$</Attribute>",
+            @"        <Attribute>^System.Diagnostics.DebuggerNonUserCodeAttribute$</Attribute>",
+            @"        <Attribute>System.Runtime.CompilerServices.CompilerGeneratedAttribute$</Attribute>",
+            @"        <Attribute>^System.CodeDom.Compiler.GeneratedCodeAttribute$</Attribute>",
+            @"        <Attribute>^System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverageAttribute$</Attribute>",
+            @"        <Attribute>^Microsoft.VisualStudio.TestPlatform.TestSDKAutoGeneratedCode.*</Attribute>",
+            @"      </Exclude>",
+            @"    </Attributes>",
+            @"    <Sources>",
+            @"      <Exclude>",
+            @"        <Source>.*\\atlmfc\\.*</Source>",
+            @"        <Source>.*\\vctools\\.*</Source>",
+            @"        <Source>.*\\public\\sdk\\.*</Source>",
+            @"        <Source>.*\\externalapis\\.*</Source>",
+            @"        <Source>.*\\microsoft sdks\\.*</Source>",
+            @"        <Source>.*\\vc\\include\\.*</Source>",
+            @"        <Source>.*\\msclr\\.*</Source>",
+            @"        <Source>.*\\ucrt\\.*</Source>",
+            @"      </Exclude>",
+            @"    </Sources>",
+            @"    <CompanyNames/>",
+            @"    <PublicKeyTokens/>",
+            @"  </CodeCoverage>",
+            @"</Configuration>");
+
+        private CodeCoverageRunSettingsProcessor processor;
+
+        private XmlDocument document;
+
+        private XmlNode ExtractNode(XmlNode node, string path)
```

Comment:
Please put all  private methods to bottom of file with tests.

Reply:
Done.